ProfitSite/src/main/java/Profit/AdItem.java

```diff
@@ -0,0 +1,135 @@
+package Profit;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class AdItem {
+
+    private String id;
```

Comment:
Generally it's a good practice to add `final` to declarations of fields that you initialize only once (like in a constructor)